src/http/error_handler.ml

```diff
@@ -189,18 +189,21 @@ let customize template (error : Error.error) =
 
 
 
-(* TODO LATER Make a nice default template. *)
 let default_template debug_dump response =
-  let response =
-    match debug_dump with
-    | None -> response
-    | Some info ->
-      response
-      |> Dream.with_body info
-      |> Dream.with_header "Content-Type" "text/plain"
-  in
-
-  Lwt.return response
+  let status = Dream.status response in
+  let code = Dream.status_to_int status
+  and reason = Dream.status_to_string status in
+  match debug_dump with
+  | None -> Lwt.return response
+  | Some s ->
+    let body =
+      let debug_dump = Dream__pure.Formats.html_escape s in
```

Comment:
I just noticed that this is probably redundant, as `<%s ... %>` inside the template already applies `html_escape`. I'll just patch it in a future 1-line (or 1-char?) commit.